pkg/pipelines/routes/routes.go

```diff
@@ -1,62 +1,63 @@
 package routes
 
 import (
-	routev1 "github.com/openshift/api/route/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/intstr"
+    routev1 "github.com/openshift/api/route/v1"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/apimachinery/pkg/util/intstr"
 )
 
 // Generate creates a route for event listener
-func Generate() routev1.Route {
-	return routev1.Route{
-		TypeMeta:   createRouteTypeMeta(),
-		ObjectMeta: createRouteObjectMeta("github-webhook-event-listener"),
-		Spec: routev1.RouteSpec{
-			To: creatRouteTargetReference(
-				"Service",
-				"el-cicd-event-listener",
-				100,
-			),
-			Port:           createRoutePort(8080),
-			WildcardPolicy: routev1.WildcardPolicyNone,
-		},
-	}
+func Generate(ns string) routev1.Route {
+    return routev1.Route{
+        TypeMeta:   createRouteTypeMeta(),
+        ObjectMeta: createRouteObjectMeta("github-webhook-event-listener", ns),
+        Spec: routev1.RouteSpec{
```

Comment:
Please make namespace the first argument so that it is consistent with other methods.